<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiStation</class>
 <widget class="QDialog" name="UiStation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="StackedView">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>49</y>
     <width>601</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TitleLabel">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>10</y>
     <width>141</width>
     <height>45</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>45</height>
    </size>
   </property>
   <property name="text">
    <string>Station Title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonNextPage">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>490</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
